Source/player.h

```diff
@@ -151,177 +151,177 @@ struct PlayerStruct {
 	bool _pLvlVisited[NUMLEVELS];
 	bool _pSLvlVisited[NUMLEVELS]; // only 10 used
 	                               /** Using player_graphic as bitflags */
-	Sint32 _pGFXLoad;
-	Uint8 *_pNAnim[8]; // Stand animations
-	Sint32 _pNFrames;
-	Sint32 _pNWidth;
-	Uint8 *_pWAnim[8]; // Walk animations
-	Sint32 _pWFrames;
-	Sint32 _pWWidth;
-	Uint8 *_pAAnim[8]; // Attack animations
-	Sint32 _pAFrames;
-	Sint32 _pAWidth;
-	Sint32 _pAFNum;
-	Uint8 *_pLAnim[8]; // Lightning spell cast animations
-	Uint8 *_pFAnim[8]; // Fire spell cast animations
-	Uint8 *_pTAnim[8]; // Generic spell cast animations
-	Sint32 _pSFrames;
-	Sint32 _pSWidth;
-	Sint32 _pSFNum;
-	Uint8 *_pHAnim[8]; // Getting hit animations
-	Sint32 _pHFrames;
-	Sint32 _pHWidth;
-	Uint8 *_pDAnim[8]; // Death animations
-	Sint32 _pDFrames;
-	Sint32 _pDWidth;
-	Uint8 *_pBAnim[8]; // Block animations
-	Sint32 _pBFrames;
-	Sint32 _pBWidth;
+	int _pGFXLoad;
+	uint8_t *_pNAnim[8]; // Stand animations
+	int _pNFrames;
+	int _pNWidth;
+	uint8_t *_pWAnim[8]; // Walk animations
+	int _pWFrames;
+	int _pWWidth;
+	uint8_t *_pAAnim[8]; // Attack animations
+	int _pAFrames;
+	int _pAWidth;
+	int _pAFNum;
+	uint8_t *_pLAnim[8]; // Lightning spell cast animations
+	uint8_t *_pFAnim[8]; // Fire spell cast animations
+	uint8_t *_pTAnim[8]; // Generic spell cast animations
+	int _pSFrames;
+	int _pSWidth;
+	int _pSFNum;
+	uint8_t *_pHAnim[8]; // Getting hit animations
+	int _pHFrames;
+	int _pHWidth;
+	uint8_t *_pDAnim[8]; // Death animations
+	int _pDFrames;
+	int _pDWidth;
+	uint8_t *_pBAnim[8]; // Block animations
+	int _pBFrames;
+	int _pBWidth;
 	ItemStruct InvBody[NUM_INVLOC];
 	ItemStruct InvList[NUM_INV_GRID_ELEM];
-	Sint32 _pNumInv;
+	int _pNumInv;
 	int8_t InvGrid[NUM_INV_GRID_ELEM];
 	ItemStruct SpdList[MAXBELTITEMS];
 	ItemStruct HoldItem;
-	Sint32 _pIMinDam;
-	Sint32 _pIMaxDam;
-	Sint32 _pIAC;
-	Sint32 _pIBonusDam;
-	Sint32 _pIBonusToHit;
-	Sint32 _pIBonusAC;
-	Sint32 _pIBonusDamMod;
+	int _pIMinDam;
+	int _pIMaxDam;
+	int _pIAC;
+	int _pIBonusDam;
+	int _pIBonusToHit;
+	int _pIBonusAC;
+	int _pIBonusDamMod;
 	/** Bitmask of staff spell */
-	Uint64 _pISpells;
+	uint64_t _pISpells;
 	/** Bitmask using item_special_effect */
-	Sint32 _pIFlags;
-	Sint32 _pIGetHit;
-	Sint8 _pISplLvlAdd;
-	Sint32 _pISplDur;
-	Sint32 _pIEnAc;
-	Sint32 _pIFMinDam;
-	Sint32 _pIFMaxDam;
-	Sint32 _pILMinDam;
-	Sint32 _pILMaxDam;
+	int _pIFlags;
+	int _pIGetHit;
+	int8_t _pISplLvlAdd;
+	int _pISplDur;
+	int _pIEnAc;
+	int _pIFMinDam;
+	int _pIFMaxDam;
+	int _pILMinDam;
+	int _pILMaxDam;
 	item_misc_id _pOilType;
-	Uint8 pTownWarps;
-	Uint8 pDungMsgs;
-	Uint8 pLvlLoad;
+	uint8_t pTownWarps;
+	uint8_t pDungMsgs;
+	uint8_t pLvlLoad;
 	bool pBattleNet;
 	bool pManaShield;
-	Uint8 pDungMsgs2;
+	uint8_t pDungMsgs2;
 	bool pOriginalCathedral;
-	Uint16 wReflections;
-	uint32_t pDiabloKillLevel;
+	uint16_t wReflections;
+	uint8_t pDiabloKillLevel;

```

Comment:
Changed by mistake?

Reply:
No, this annoyance happened as part of cleaning up the logic that manipulates it. Suppose it could have gone in as a separate commit.